recipe/meta.yaml

```diff
@@ -7,7 +7,7 @@ package:
 
 source:
   url: https://github.com/pymc-devs/{{ name }}/archive/refs/tags/rel-{{ version }}.tar.gz
-  sha256: 468eb208e5a9fe62530752a354f48f739442b45811797e084ea432809ca5f99c
+  sha256: a1c436c67376436d2030742434cae7ff450a07c207c352f26ac958b0fb60142e
 
 build:
   number: 0
```

Comment:
Could add skip py<38 to respect the upstream (but I know we do not build for versions less than that anymore, atm)
https://github.com/pymc-devs/pytensor/blob/rel-2.12.3/pyproject.toml#L15

Reply:
will do.

Reply:
oh actually it's a little pointless to do this here because this will not be merged to main, as this is going to land in a versioned feedstock. But I will bear it in mind for the future.